File: euler9.py
import math
def euler9 ():
  a = range(1,1001)
  b = range(1, 1001)
  c = math.sqrt(1000)
  for a1 in a:
    for b1 in b:
      
      solve = a1**2 + b1**2
      solvesqr = math.sqrt(solve)
      solve1 = a1+ b1 + solvesqr
      
      
      if solve1 == 1000:
        solve2 = a1*b1*solvesqr
        print("a: {} b: {} c: {}".format(a1, b1, solvesqr))
        print( f"{a1} x {b1} x {solvesqr} = {solve2}")
  
euler9()

File: euler9.java
import java.lang.Math;

public class euler9{
  public static void main(String[] args){
    
    for(int a = 1; a < 1000; a++){
      
      for(int b = 1; b < 1000; b++){
        
        double a2 = Math.pow(a, 2);
        double b2 = Math.pow(b, 2);
        double c = a2 + b2;
        double csqr = Math.sqrt(c);
        double sum = a + b + csqr;
        // System.out.println(""+sum);
        if (sum == 1000.0){
          double prod = a*b*csqr;
          System.out.println("" + prod);
          
        }
        
      }
      
    }
    
  }
}